README.md

```diff
@@ -26,9 +26,10 @@
 ```
 -->
 ```html
-<vaadin-custom-field>
-  ...
-</vaadin-custom-field>
+<vaadin-custom-field label="Choose appointment time">
```

Comment:
Please update the screenshot to match the updated demo
Also, "Appointment time" here is enough, too